R/plot.vpts.R

```diff
@@ -220,7 +220,7 @@ plot.vpts <- function(x, xlab = "time", ylab = "height [m]", quantity = "dens",
     if ("ylim" %in% names(args)) {
       h.barbs <- seq(min(args$ylim), max(args$ylim), length.out = barbs_height)
     } else {
-      h.barbs <- seq(x$heights[1], tail(x$heights, 1), length.out = barbs_height)
+      h.barbs <- seq(x$heights[1], tail(x$heights, 1) + interval, length.out = barbs_height)
```

Comment:
`interval/2` here as well?

Reply:
I think not since this is only giving the start and end of the sequence over which the heights are distributed. Now is it the same as the ylim. Alternatively It could be:
```r
       h.barbs <- seq(x$heights[1] + interval/2, tail(x$heights, 1) + interval/2, length.out = barbs_height) 
```
This would start and end the barbs midway the first and last interval